.github/workflows/build-toolchain.yml

```diff
@@ -455,73 +455,164 @@ jobs:
               "include": [
                 {
                   "arch": "amd64",
+                  "cpu": "x86_64",
                   "os": "Windows",
+                  "platform": "Windows",
+                  "triple": "x86_64-unknown-windows-msvc",
+                  "triple_no_api_level": "x86_64-unknown-windows-msvc",
                   "cc": "cl",
                   "cflags": "${{ steps.context.outputs.WINDOWS_CMAKE_C_FLAGS }}",
                   "cxx": "cl",
                   "cxxflags": "${{ steps.context.outputs.WINDOWS_CMAKE_CXX_FLAGS }}",
                   "swiftflags": "${{ steps.context.outputs.WINDOWS_CMAKE_Swift_FLAGS }}",
-                  "extra_flags": "-D CMAKE_SYSTEM_PROCESSOR=AMD64"
+                  "cmake_linker_flags": "-D CMAKE_EXE_LINKER_FLAGS=\"${{ steps.context.outputs.WINDOWS_CMAKE_EXE_LINKER_FLAGS }}\" -D CMAKE_SHARED_LINKER_FLAGS=\"${{ steps.context.outputs.WINDOWS_CMAKE_EXE_LINKER_FLAGS }}\"",
+                  "cmake_ndk_flags": "",
+                  "extra_flags": "-D CMAKE_SYSTEM_PROCESSOR=AMD64",
+                  "build_cc": "${HostToolchain}/usr/bin/clang-cl.exe",
+                  "build_cxx": "${HostToolchain}/usr/bin/clang-cl.exe",
```

Comment:
These are confusing. The build C/C++ compilers change between phases.

Reply:
That was already the case prior to these changes. This is making the compiler used (system vs. built) more explicit.
Should we use the same compiler everywhere? I don't know if the long-term goal is to use `clang-cl` or MSVC `cl`.

Reply:
> That was already the case prior to these changes

Sure, the compiler changing between phases was already the case, but we didn't label clang as the build compiler and cl as something else. This shouldn't be called `build_cc`, unless you want to define this path 25-75 times, once for each of the repository/target case.

Reply:
I have removed them and am now setting the compiler manually before running configure.